REQUIRE

```diff
@@ -1,3 +1,3 @@
-julia 0.6.0-pre
-LearnBase 0.1.3 0.2.0
+julia 0.7-beta
+LearnBase 0.1.3 0.3.0
```

Comment:
maybe do

```
LearnBase 0.2.0 0.3.0
```

Reply:
Yep, I left it for now so that the CI will run.  I'll bump the lower bound when the LearnBase release gets into metadata.

Reply:
right, good point